Multisig/Features/Connect to Web/UI/List/WebConnectionsViewController.swift

```diff
@@ -156,6 +156,10 @@ class WebConnectionsViewController: UITableViewController, ExternalURLSource, We
             UIContextualAction(style: .destructive, title: "Disconnect") {  [weak self] _, _, completion in
                 guard let `self` = self else { return }
                 let alertController = DisconnectionConfirmationController.create(connection: connection)
+                if let popoverPresentationController = alertController.popoverPresentationController {
+                    popoverPresentationController.sourceView = tableView
```

Comment:
here, too, maybe tableView.cellForRow(at:) would be enough and no need to set the sourceRect?

Reply:
This i have to test on a real iPad. On the mac the handover to metamask doesnt work. Even though I have metamask for iphone installed.

![Uploading image.png…]()

Reply:
Is it possible to install MetaMask in the simulator? Or any app from the App Store?

Reply:
Metamask communication works fne on an iPad